converter/dicom-cast/src/Microsoft.Health.DicomCast.TableStorage/Features/Storage/TableExceptionStore.cs

```diff
@@ -78,5 +81,33 @@ public async Task StoreExceptionToTable(ChangeFeedEntry changeFeedEntry, Excepti
                 throw;
             }
         }
+
+        /// <inheritdoc/>
+        public async Task WriteRetryableExceptionAsync(ChangeFeedEntry changeFeedEntry, int retryNum, Exception exceptionToStore, CancellationToken cancellationToken)
```

Comment:
These methods are very similar. Is there a way to use a helper function to share some functionality?

Reply:
As discussed off line will leave as is due to not enough overlap of functionality.